static/mobile.css

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-device-width: 750px) {
```

Comment:
The 750px limit is arbitrary, it's just to detect if the device is mobile or not. Would bumping it to 800px solve the tablet problem?